storybook/stories/Layout/AppHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { AppHeader } from "../../../src/components/Layout/AppHeader";
+import { View, Text } from "react-native";
+import { size, color } from "../../../src/common/styles";
+import { AppMode } from "../../../src/context/config";
+import { createAppContainer } from "react-navigation";
+import { createDrawerNavigator } from "react-navigation-drawer";
+
+const reactNavigationDecorator = (story: any): any => {
+  const Screen = (): any => story();
+  const Navigator = createAppContainer(
+    createDrawerNavigator(
+      { Screen },
+      {
+        drawerPosition: "right",
+        drawerType: "slide",
+      }
+    )
+  );
+  return <Navigator />;
+};
+
+const appHeaderElements = (): any[] => {
+  return [
+    <>
+      <Text>Production App Header</Text>
+      <AppHeader />
+    </>,
+    <>
+      <Text>Staging App Header</Text>
+      <AppHeader mode={AppMode.staging} />
+    </>,
+  ];
+};
+
+storiesOf("Layout", module)
+  .addDecorator(reactNavigationDecorator) // decorator is used to wrapper of the story
```

Comment:
Why is there a need for the navigation slider?

Reply:
the navigation props is used in `AppHeader` so it should wrapper with navigator , otherwise we got a render error